entry/vexflow-font-gonville.ts

```diff
@@ -1,11 +1,11 @@
 // [VexFlow](https://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
 // MIT License
 
-import { GonvilleFont } from '../src/fonts/gonville_glyphs';
+import { GonvilleSmuflFont } from '../src/fonts/gonville_glyphs';
```

Comment:
I did not want to manually change the result of the conversion and I got the following feedbak from the smufl group:
___
De: Elbsound.studio <info@elbsound.studio>
Date: mar, 6 dic 2022 a las 9:23
Subject: Re: Gonville SMuFL
To: Rodrigo Vilar <rvilarl@gmail.com>


Hi Rodrigo,

good  job on Gonville!

However it would be great if you could give it a clear and unique name, like Gonville SMuFl.
Some notation software (e.g. Finale) also use the original Gonville font and it leads to problems when there are two completely different fonts that use the identical name.

As far as I know the original Gonville font from 2012 was released un der SIL OFL license.
That means:
1.) You have to publish it under the SIL OFL license too (and not change the license to MIT)
2.) Releasing it under SIL OFL means that you need a unique font name anyway.
As you can read in the original Gonville SIL OFL license text: "...with Reserved Font Name Gonville".

I am the developer of the [Elbsound Music Fonts](https://elbsound.studio/elbsound-music-font-package-for-finale.php) (which also includes Gonville for Finale) and the [Music Font Comparison](https://elbsound.studio/music-font-comparison.php?font=Gonville) and I also run the website [www.music-fonts.com](http://www.music-fonts.com/).

Jan Angermüller
Elbsound.studio
___